vendors/circom/README.md

```diff
@@ -46,14 +46,16 @@ bazel build -c opt --config macos_x86_64 //:prover_main
 bazel-bin/prover_main -h
 Usage:
 
-bazel-bin/prover_main zkey wtns proof public [OPTIONS]
+bazel-bin/prover_main zkey wtns proof public -n [OPTIONS]
```

Comment:
```suggestion
bazel-bin/prover_main zkey wtns proof public [OPTIONS]
```

Reply:
Implemented this change in the current commit